arctic/store/_pandas_ndarray_store.py

```diff
@@ -195,6 +196,17 @@ def read(self, arctic_lib, version, symbol, read_preference=None, date_range=Non
             item = self._daterange(item, date_range)
         return item
 
+    def get_data_info(self, version):
```

Comment:
I agree get_info is nicer, but its already used by the parent class and other storage types for something different.